mvrx/src/main/kotlin/com/airbnb/mvrx/mock/MockableMvRxStateStore.kt

```diff
@@ -0,0 +1,113 @@
+package com.airbnb.mvrx.mock
+
+import com.airbnb.mvrx.RealMvRxStateStore
+import com.airbnb.mvrx.ScriptableMvRxStateStore
+import com.airbnb.mvrx.ScriptableStateStore
+import com.airbnb.mvrx.mock.MockBehavior.StateStoreBehavior
+import io.reactivex.Observable
+import io.reactivex.schedulers.Schedulers
+
+interface MockableStateStore<S : Any> : ScriptableStateStore<S> {
```

Comment:
Is this scriptable state store needed any more? It seems equivalent to using the block action api and the freeze state call. There seems to be a lot of state stores, and confusing to track all their different purposes.